src/main/java/wolox/training/models/Book.java

```diff
@@ -0,0 +1,113 @@
+package wolox.training.models;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.Id;
+
+@Entity
+public class Book {
+
+    @Id
+    private String isbn;
```

Comment:
For this model we are going to use an id apart from the isbn, we also need a strategy to generate successive id.

Reply:
Can I use the `@GeneratedValue(strategy = GenerationType.AUTO)` annotation o maybe we must use a `sequence`?

Reply:
**_@GeneratedValue(Strategy = GenerationType.AUTO )_** is ok